content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,205 @@
+
+This rule adds the activity status of the lead to the report.
+
+.. note::
+  Active status is an important filter to include when creating a lead distribution report because
+  it includes **all** leads regardless of won/lost or active/inactive status in the report. This
+  provides a comprehensive view of all the leads assigned to each sales member.
+
+Group by
+~~~~~~~~
+
+Once all filters are set, click the :guilabel:`Add` button to add these filters to the search bar.
+To have the report grouped appropriately, click the :icon:`fa-caret-down` :guilabel:`(down caret)`
+icon, to the right of the search bar, and click :guilabel:`Salesperson` in the :guilabel:`Group
+By` section. All results are now grouped by the salesperson assigned to each lead.
+
+Once the rules for the filter are set, click the purple :guilabel:`Confirm` button at the bottom of
+the pop-up menu to save the custom filter and close the pop-up menu.
+
+The :guilabel:`Pipeline Analysis` dashboard is now displayed again with each filter rule in the
+search bar.
+
+Click the :icon:`fa-area-chart` :guilabel:`(Graph)` icon, to the right of the
+search bar, to view the report as a bar chart. Alternatively, click the :icon:`oi-view-list`
+:guilabel:`(List)` icon to view leads in a grouped list.
+
+.. tip::
+  To save the filter so it can easily be re-applied, click the :guilabel:`Save current search`
+  button in the :guilabel:`Favorites` section of the search bar drop-down menu.
+
+  Next, type a name for the filter in the text box below. Check the :guilabel:`Shared` checkbox to
+  have the filter shared with any user with access to the pipeline. Finally, click the purple
+  :guilabel:`Save` button below to save the filter.
+
+  The filter will now appear with the name it was given under the :guilabel:`Favorites` section of
+  the drop-down menu and can be re-applied by clicking on it.
+
+Filter for quality leads
+------------------------
+
+The following additional conditions are provided as an example of a *good*, but *not comprehensive*,
+set of rules for finding quality leads. These filters should be applied on top of the
+:ref:`crm/track_leads/essential-filters` in the order specified to achieve a heavily-detailed
+filter.
+
+- **Referred-by:** Filter for referrals, such as by appointment or sales member.
+
+- **Source:** Filter for specific source UTMs, such as Facebook or LinkedIn.
+
+- **Notes:** Filter for internal notes.
+
+- **Tags:** Filter for categorical tags.
+
+- **Email:** Filter for specific email domains, such as gmail.com or yahoo.com.
+
+- **Salesperson:** Filter for leads associated with certain sales members.
+
+These conditions can be added, removed, or modified to best fit the desired information in the
+report.
+
+.. seealso::
+  `Additional rules for quality leads <https://www.odoo.com/documentation/17.0/applications/sales/
+  crm/track_leads/quality_leads_report.html#add-rules-for-quality-leads>`_
+
+.. seealso::
+   :doc:`../../../essentials/search`
```

Comment:
No need to have two seealso's, can just do a bullet list in a single seealso :)

Secondly, instead of using an external link formatting to reference a specific section within a doc, you can use the `:ref:` (reference) role, to reference a custom anchor in another document. 

Custom anchors are _global_ throughout the documentation, so once a custom anchor is created, it can be referenced throughout any doc just by using the custom anchor's name in a `:ref:` tag.

More info can be found here: https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#custom-anchors
```suggestion
   - :ref:`quality_leads_report/add-quality-rules`
   - :doc:`../../../essentials/search`
```